src/cmd.ts

```diff
@@ -109,6 +109,19 @@ async function run() {
 				"--notification-webhook-url <url>",
 				"cross-seed will send POST requests to this url with a JSON payload of { title, body }",
 				fileConfig.notificationWebhookUrl
+			)
+			.addOption(
+				new Option(
+					"-f, --fuzzy-size <decimal>",
```

Comment:
Commander requires consistent naming, so an option called `--bobs-your-uncle` in the commander config would then become `bobsYourUncle` in the runtimeConfig. In order for the runtimeConfig to correctly pick up this option, it should be named `--fuzzy-size-threshold`. I also would prefer not to have a short flag called `-f` because `-f` sort of has its own meaning. 
```suggestion
					"--fuzzy-size-threshold <decimal>",
```